src/forms/FormActions.js

```diff
@@ -103,12 +103,30 @@ export const COPY_FORM = 'COPY_FORM';
  * Utils
  */
 
-const getInit = (body, method) => ({
-  method: method || 'POST',
-  headers: new Headers({ 'Accept': 'application/json', 'Content-Type': 'application/json' }),
-  mode: 'cors',
-  body: body ? JSON.stringify(body) : undefined
-});
+const getInit = (method, body, token = '') => {
+  const init = {
+    method: method || 'POST',
+    headers: new Headers({
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'Authorization': token
+    })
+  };
+
+  if (method !== 'GET' && body) init.body = JSON.stringify(body);
+
+  return init;
+};
+
+const handleResp = res => {
+  if (res.status === 401) {
```

Comment:
@wyattjoh said he'd only be returning a 401. Unless it blows up, of course. Then it'll be 500